app/uk/gov/hmrc/trackingconsentfrontend/assets/test/interfaces/optimizelyCommunicatorFactory.spec.ts

```diff
@@ -0,0 +1,71 @@
+import optimizelyCommunicatorFactory from '../../src/interfaces/optimizelyCommunicatorFactory'
+
+describe('sendPreferences', () => {
+  let testScope
+  beforeEach(() => {
+    testScope = { window: {} }
+    testScope.preferenceCommunicator = optimizelyCommunicatorFactory(testScope.window)
+    testScope.userPreferences = { getPreferences: jest.fn() }
+    testScope.userPreferences.getPreferences.mockReturnValue({
+      measurement: false,
+      marketing: false,
+      settings: false
+    })
+  })
+
+  it('should send measurement, marketing and settings when all tracking is allowed', () => {
```

Comment:
Is that description from the gtm one?

Reply:
Good spot, updated test description and slightly re-ordered tests for readability.